GIT_AND_GITHUB.md

```diff
@@ -89,6 +77,13 @@ discuss other alternatives. Of course, this means you are putting yourself at ri
 of sharing your knowledge and/or learning some new stuff, so please be careful not
 to end up being even more awesome than you are.
 
+If everything is fine feel free to merge the pull request. Avoid merging before receiving
```

Comment:
But that doesn't apply if you submit a PR to Alamofire for example, I think this paragraph implies:

- You own the repo and made the PR
- You don't own the repo, so this Reviewing PR section doesn't apply for you because you don't have merge access